libs/crypto/src/mcl_dkg.cpp

```diff
@@ -407,14 +407,14 @@ bn::Fr SignatureAggregationCoefficient(PublicKey const &             notarisatio
  * @return Pair consisting of aggregate signature and a vector indicating who's signatures were
  * aggregated
  */
-std::pair<Signature, std::vector<bool>> ComputeAggregateSignature(
+AggregateSignature ComputeAggregateSignature(
     std::unordered_map<uint32_t, Signature> const &signatures,
     std::vector<PublicKey> const &                 public_keys)
 {
   Signature aggregate_signature;
   aggregate_signature.clear();
-  std::vector<bool> signers;
-  signers.resize(public_keys.size(), false);
+  SignerRecord signers;
+  signers.resize(public_keys.size(), 0);
```

Comment:
Worth thinking about using a bitset here if the signer record is serialized and stored for all time

Reply:
I looked into the bitset but it seemed to require you to know the number of bits required at compile time.